api/Updated-Metrics.md

```diff
@@ -0,0 +1,368 @@
+# Updates for Metrics
+
+This is an update for the existing metrics document, which is being left in place for now as a point of comparison.
+
+## Assumed data
+
+In the following we assume that we have variables of the following form defined:
+
+```python
+y_true = [0, 1, 0, 0, 1, 1, ...]
+y_pred = [1, 0, 0, 1, 0, 1, ...]
```

Comment:
since we're dealing with metrics, should we separate floating point predictions (`decision_function` or `predict`) from discrete ones? (assuming some metrics need soft values and some work with hard values)

Maybe simply add :

```suggestion
# or soft values, depending on what the underlying metric expects
y_pred = [1, 0, 0, 1, 0, 1, ...]
```

Reply:
I don't think that we want separate `group_summary()` functions for different types of underlying metrics, but I agree about the need to highlight that we just slice up the `y_pred` list regardless of the types of its items, if that's what you meant. I was trying to write that below, but I'll make it more explicit.